mypy/test/data.py

```diff
@@ -6,14 +6,14 @@
 from os import remove, rmdir
 import shutil
 
-from typing import Callable, List, Tuple
+from typing import Callable, List, Tuple, Any
 
 from mypy.myunit import TestCase, SkipTestCaseException
 
 
 def parse_test_cases(
         path: str,
-        perform: Callable[['DataDrivenTestCase'], None],
+        perform: Callable[[Any, 'DataDrivenTestCase'], None],
```

Comment:
Hmm. What's up with this `Any` type? Always preferable to avoid those where possible -- where it isn't, it's good to explain in a comment somewhere what the obstacle is.